src/plugin/managers/lifecycle.spec.ts

```
@@ -0,0 +1,498 @@
+
+      await manager.activatePlugin(VALID_PLUGIN_ID);
+
+      expect(consoleSpy).toHaveBeenCalledWith(
+        expect.stringContaining('Failed to register plugin'),
+        expect.anything()
+      );
+      consoleSpy.mockRestore();
+    });
+
+    it('handles missing manifest during activation (skips extension registration)', async () => {
+        const managerNoManifest = createManager();
```

ESLint

[error] Delete `··`

(prettier/prettier)